website/docs/gator.md

```diff
@@ -62,6 +63,23 @@ Or you can specify both as flags:
 gator test -f=my-manifest.yaml -f=templates-and-constraints/
 ```
 
+The `--image` flag specifies a content addressable OCI artifact containing
+policy files. The image(s) will be copied into the local filesystem in a
+temporary directory, the location of which can be overridden with
+the `--tempdir`
+flag. Only files with the aforementioned extensions will be processed. For
+information on how to create OCI policy bundles, see
+the [Bundling Policy into OCI Artifacts](#bundling-policy-into-oci-artifacts)
+section.
+
+For example, to test a manifest (piped via stdin) against an OCI Artifact
+containing policies:
+
+```
+cat my-manifest.yaml | gator test --image=localhost:5000/gator/template-library:v1 \
+  --image=localhost:5000/gator/constraints:v1 
```

Comment:
this is assuming parameters are pre-configured? seems like there's no way to pass in parameters?

Reply:
Do OCI images have a way of being parameterized?

I'm guessing actual parameterized bundles may need something different and/or on top of containerization.

Reply:
I don't think so, that's why I think it would be good to define use cases for how we think this should be used in CI. Should we mark this feature as alpha since gator is going to be beta in next release?

Reply:
I added a note in the docs that this feature is alpha.